benchmarks/CMakeLists.txt

```diff
@@ -79,3 +79,10 @@ set(CUDA_STREAM_POOL_BENCH_SRC
     "${CMAKE_CURRENT_SOURCE_DIR}/cuda_stream_pool/cuda_stream_pool_bench.cpp")
 
 ConfigureBench(CUDA_STREAM_POOL_BENCH "${CUDA_STREAM_POOL_BENCH_SRC}")
+
+# multi stream allocations
```

Comment:
With https://github.com/rapidsai/rmm/pull/842 merged, you should update the benchmark/CMakeLists.txt to match the new style.